### PR TITLE
:tada: Report button added to home

### PR DESCRIPTION
**Pull request by vdd5**

### CODE REVIEW

File: src/main/resources/templates/home.html [hunk 1]
```diff
@@ -27,5 +28,11 @@ <h1>Get Vaccinated</h1>
 </div>
 
  <script type="text/javascript" th:src="@{/js/home.js}"></script>
+  <script type="text/javascript">
+    function redirect() {
+      window.open(' https://vaers.hhs.gov/ ', '_blank');
+    }
+  </script>
+  </script>
```
Comment (rsolomo) [inline src/main/resources/templates/home.html:36]: There is an extra closing `</script>` tag here.

File: src/main/resources/templates/home.html [hunk 2]
```diff
@@ -10,6 +10,7 @@
 <main>
   <h1>Get Vaccinated</h1>
   <p><button th:onclick="showWarning()">Schedule a Vaccination</button></p>
+  <p><button onclick="redirect()">Report side effects</button></p>
```
Comment (rsolomo) [inline src/main/resources/templates/home.html:13]: This could be more simply implemented as an `<a>` tag that links to the destination page.

Comment (rsolomo) [reply src/main/resources/templates/home.html:13]: If you want to leave it as a button, to simplify having consistent styling, that's fine too. The extra closing `</script>` is the primary concern.

**Comment by vdd5** [comment]
@rsolomo I've resolved all the related issues please merge.

**Pull request merged**